etna/models/nn/deepar.py

```diff
@@ -69,7 +70,7 @@ def __init__(
         self.rnn_layers = rnn_layers
         self.dropout = dropout
 
-    def _from_dataset(self, ts_dataset: TimeSeriesDataSet) -> DeepAR:
+    def _from_dataset(self, ts_dataset: TimeSeriesDataSet) -> LightningModule:
```

Comment:
Formally, it's true but it's less helpfull, imho

Reply:
I can set `# type: ignore` in this case